types/workgrid-toolbar/index.d.ts

```diff
@@ -14,12 +14,15 @@ export interface WorkgridToolbarOptions {
     // Optional
     confirmUser?: boolean | false;
     debug?: boolean | false;
+    deferDisplay?: boolean | false;
```

Comment:
I see that the types here are redundant.
This needs to be just boolean since boolean | false = boolean